vterm/ops.go

```diff
@@ -160,6 +160,18 @@ func (v *VTerm) shiftCursorY(diff int) {
 	v.setCursorPos(v.Cursor.X, v.Cursor.Y+diff)
 }
 
+func (v *VTerm) setChar(x, y int, r rune, style ecma48.Style) {
+	if x >= v.w {
+		return
+	}
```

Comment:
Instead of truncating, should we also log that there was some sort of overflow?

Reply:
Ah, so user input isn't yet fully verified before being passed to this function. Once we move some bounds checking closer to stdout.go, we'll be able to distinguish between an internal error versus malformed user input